docs/superpowers/plans/2026-07-29-observer-node-mode.md

```
@@ -0,0 +1,988 @@
+            // earlier-firing genesis-hash check passes and the later
+            // periodic-sync check is the one that surfaces.
+            nc.consensus.get_mut().expected_genesis_hash = Some(H256::zero());
+```
+
+If these two tests assert on error-message substrings further down, update the expected substring to match the new wording from Step 4.
+
+- [ ] **Step 6: Update the non-Rust config files and docs**
+
+Set `node_mode = "Miner"` in each of:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)